lib/internal/test_runner/harness.js

```diff
@@ -77,7 +77,9 @@ function setup(root) {
     createProcessEventHandler('unhandledRejection', root);
 
   const exitHandler = () => {
-    root.postRun();
+    const msg = 'There is no more asynchronous activity queued in the event loop, ' +
```

Comment:
I would rephrase it to:
> Exit on async queue test error

Reply:
Plus, event loop always can be busy with something else, so it's getting even more confusing with current change 🙂

Reply:
> Plus, event loop always can be busy with something else

IIUC if the event loop was busy, the test_runner wouldn't exit and we wouldn't need to display this error message.

Deno output for this use case is `error: Promise resolution is still pending but the event loop has already resolved.`. Seems quite clear to me, wdyt?

Reply:
@aduh95 is this an error deno throws in any case where there is a unresolved promise, or specifically when running tests?

Reply:
It's the result of `Deno.test(() => new Promise(() => {});`.

FWIW in case of an unfinished TLA, it outputs `error: Module evaluation is still pending but there are no pending ops or dynamic imports. This situation is often caused by unresolved promises.`. Unresolved promises elsewhere are completely ignored.

Reply:
updated the wording